pkg/register/authorization.go

```diff
@@ -18,8 +17,7 @@ import (
 )
 
 const (
-	regexpBasicAuth = `^[Bb]asic (.+)$`
-	regExpRecaptcha = `^([\w\d]+):secret=(.+),token=(.+)$`
+	regexpRecaptchaToken = `^[\w\d-]+$`
```

Comment:
Do we have a max size for this token ?

Reply:
We had an example with almost 480 characters but we don't really have details about this token.